marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -33,7 +34,8 @@ const registeredArtifactTypes = {
     flow: Flow,
     stepDefinition: StepDef,
     mapping: Mapping,
-    matching: Matching
+    matching: Matching,
+    custom: Custom
```

Comment:
I think we need "customStep" and "CustomStep" here as well. "Custom" is just too vague by itself for the reason mentioned above (custom step, custom module, custom hook).

Reply:
"custom" here refers to the step definition type. I have retained that. It is custom:CustomStep in the latest commit